fedora/python-ogr.spec

```
@@ -52,6 +52,9 @@ rm -rf %{srcname}.egg-info
 
 
 %changelog
+* Tue Jan 04 2022 Tomas Tomecek <ttomecek@redhat.com> - 0.34.0-1
+- 0.34.0 upstream release
```

```suggestion
- New upstream release 0.34.0
```

pita, but to be consistent

---

I need to also change that in my shell history since I just rerun the command :D

---

well… we have it like this here, and in `packit` it's the other way :D